client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -293,6 +293,25 @@
             });
             item._locked = true;
         };
+
+        /**
+        * Link an item for takes.
+        * @param {Object} item : Target Item
+        * @param {string} link_id: Optional. If not provider it returns the new Linked item.
+        * @param {string} desk: Optional. Desk for newly create item.
```

Comment:
Tip: Parameters can also be [marked as optional](http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values) by wrapping them into angular brackets, and then the docs generator takes care of the `optional`notion.